sdk/src/obj_worker.ts

```diff
@@ -0,0 +1,10 @@
+import { expose } from 'comlink';
+
+const obj = {
+    counter: 0,
+    inc() {
+        this.counter++;
+    },
+};
+
+expose(obj);
```

Comment:
This is an alternative worker correct for illustrative purposes to show the usage of the workers? Per our last conversation - maybe we have a `workers` directory with the other worker file + one other worker focused maybe on record scanning and make it abundantly clear in the worker module docs that these are base patterns for using the worker as an api.